app/domain/__init__.py

```
@@ -6,6 +6,7 @@
 from .competency_extraction_pipeline_execution_dto import (
     CompetencyExtractionPipelineExecutionDTO,
 )
+from .inconsistency_check_pipeline_execution_dto import InconsistencyCheckPipelineExecutionDTO
```

Ruff (0.8.2)

`.inconsistency_check_pipeline_execution_dto.InconsistencyCheckPipelineExecutionDTO` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)